lld/ELF/ScriptLexer.cpp

```diff
@@ -90,6 +90,22 @@ std::string ScriptLexer::getCurrentLocation() {
   return (filename + ":" + Twine(getLineNumber())).str();
 }
 
+std::string ScriptLexer::joinTokens(size_t begin, size_t end) {
```

Comment:
https://llvm.org/doxygen/classllvm_1_1mapped__iterator__base.html

Reply:
This function may not be needed once the Lexer can be fully stateful one. I plan to keep this for temporary resolution for printing out StringRef from `struct Token`